volatility/framework/plugins/windows/envars.py

```diff
@@ -0,0 +1,161 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+# Creator: Aviel Zohar (memoryforensicsanalysis@gmail.com) # Fix vol2 plugin to vol3
+
+from typing import Callable, List, Generator, Iterable, Dict
+from volatility.framework import renderers, interfaces, objects, exceptions
+from volatility.framework.configuration import requirements
+from volatility.framework.objects import utility
+from volatility.plugins.windows import pslist
+from volatility.framework.symbols.windows.extensions.registry import RegValueTypes
+import volatility.plugins.windows.registry.hivelist as hivelist

```

Comment:
Not a big deal, but to match everything else, this would be nicer as `from volatility.plugins.windows.registry import hivelist`.